src/main/java/org/apache/commons/imaging/color/ColorCieLch.java

```diff
@@ -19,10 +19,14 @@
 /**
  * Represents a color in the CIELCH color space.
  *
- * <p>Contains the constant values for black, white, red,
- * green, and blue.</p>
+ * <p>
+ * Contains the constant values for black, white, red,
+ * green, and blue.
+ * </p>
+ * Changes: H renamed to h.
```

Comment:
The changes must be documented in a JIRA issue and linked to the changelog (i.e. the JIRA number is added to src/changes/changes.xml later, used to generate the changes report from the project site).

Reply:
> The changes must be documented in a JIRA issue and linked to the changelog (i.e. the JIRA number is added to src/changes/changes.xml later, used to generate the changes report from the project site).

I don't have access to the JIRA. Would you mind to do that for me, given my comments in this PR?

Reply:
Sure, will create it later and link it here :-)

Reply:
> Sure, will create it later and link it here :-)

Splendid, thank you!

Reply:
Not a problem. But this needs to be removed from this file anyway in the meantime :+1: I will create the issue later once I think of a good title/description.